app/views/questionnaire.py

```diff
@@ -107,7 +108,9 @@ def get_block(routing_path, schema, metadata, answer_store, block_id):
 
     transformed_block = transform_variants(block, schema, metadata, answer_store)
 
-    placeholder_renderer = PlaceholderRenderer(answer_store=answer_store, metadata=metadata)
+    placeholder_renderer = PlaceholderRenderer(locale=flask_babel.get_locale(),
```

Comment:
Locale could be passed as a string e.g. `en` or `en_GB` that Babel will understand. This would mean the external interface of the Placeholder renderer wouldn't expose the use of Babel for translations.